cmd/schedule-builder/model/model.go

```diff
@@ -32,6 +32,7 @@ type PreviousPatches struct {
 // Schedule struct to define the release schedule for a specific version
 type Schedule struct {
 	Release            string            `yaml:"release"`
+	ReleaseDate        string            `yaml:"releaseDate"`
 	Next               string            `yaml:"next"`
```

Comment:
https://github.com/kubernetes/website/pull/36609 adds more context to the `next` to make it homogenous with PreviousPatches structure.